### ISSUE TITLE
[Feature Request]: Encrypted local files

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#3223]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

Usecase: Your sharing a PC with a whole household and just dont want someone to snoop on your stuff.

### Proposed Solution

Encrypt all local files listed in https://docs.freetubeapp.io/usage/data-location/#notable-files

### Alternatives Considered

None

### Issue Labels

new feature

**Comment by absidue** [comment]
The problem description is already solved at a operating system and file system level, mutliple user accounts.

If you really do need to encrypt those files, then you would need to enter the decryption key every time you open FreeTube (this would need to be another password separate from the settings password).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
We cant assume that everybody is use multiple accounts. I personally know households that use one account.

**Comment by tciesla** [comment]
I agree with @efb4f5ff-1298-471a-8973-3d47447115dc , for example in my household we just share one account. I see another potential solution here: isn't it possible to run multiple instances of FreeTube app with different user data path? These instances may be placed on different workspaces (depends on desktop environment) or minimized/maximized.
